src/libraries/System.Diagnostics.PerformanceCounter/src/System/Diagnostics/PerformanceData/CounterSet.cs

```diff
@@ -256,7 +256,7 @@ public CounterSetInstance CreateCounterSetInstance(string instanceName)
                             {
                                 throw Status switch
                                 {
-                                    (uint)Interop.Errors.ERROR_ALREADY_EXISTS => new ArgumentException(SR.Format(SR.Perflib_Argument_CounterSetAlreadyRegister, _counterSet), "CounterSetGuid"),
+                                    (uint)Interop.Errors.ERROR_ALREADY_EXISTS => new ArgumentException(SR.Format(SR.Perflib_Argument_CounterSetAlreadyRegister, _counterSet), nameof(instanceName)),
```

Comment:
Similar question

Reply:
Similar answer :), there is no argument like "CounterSetGuid" and the exception message `"CounterSet '{0}' already registered."` looks pretty descriptive, but please let me know if we need to suppress instead

Reply:
I think this is same, maybe better to be InvalidOperationException

Reply:
It is definitely more appropriate to be an InvalidOperationException; but if we change the exception thrown by the same condition we should flag it as a breaking change and put it in the breaking change documentation for the release.

Reply:
The same doc applies to this API, by which it should be InvalidOperationException, in this case it still needs to be added to the breaking change? I guess yes even the old exception was not documented
 ![image](https://user-images.githubusercontent.com/45579687/81208622-34a67280-8f84-11ea-8641-08712bd0f555.png)

Reply:
> in this case it still needs to be added to the breaking change

Yeah, it's not whether or not the API documentation needs to change, it's about whether someone may have written a try/catch where they tried to handle this case and now their catch may not hit.